platform/macos/macos.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,8 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		30E5781B1DAA857E0050F07E /* NSImage+MGLAdditions.h in Headers */ = {isa = PBXBuildFile; fileRef = 30E578141DAA7D920050F07E /* NSImage+MGLAdditions.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
This header’s visibility should be Project, not Public. Making it public would expose C++ syntax to pure Objective-C projects using the SDK.